lib/modules/platform/github/pr.ts

```diff
@@ -12,7 +12,6 @@ function getPrApiCache(): ApiCache<GhPr> {
   const repoCache = getCache();
   repoCache.platform ??= {};
   repoCache.platform.github ??= {};
-  delete repoCache.platform.github.prCache;
```

Comment:
why this?

Reply:
This field was last used 1.5 years ago, so hopefully we've migrated all cache instances since then.